scripts/imm_utils.sh

```diff
@@ -20,7 +20,7 @@ IMM_CONFIG_PATH=~/.immconfig
 get_token() {
 	user=$1
 	echo "Get $user token"
-	TOKEN=`python -W ignore ../tests/management_api_tests/authenticate.py $user`
+	TOKEN=`python ../tests/management_api_tests/authenticate.py $user`
```

Comment:
when we use `ignore` option, we still get errors, because required `enum` package is loaded after requests, what causes error about missing package.
To avoid this i moved ignoring errors directly to authenticate.py script.